Uploadcare.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'Uploadcare'
-  s.version          = '0.8.2'
+  s.version          = '0.8.3'
```

Comment:
Do we have any breaking changes? Update looks big. Consider 0.9.0? (And add-ons will bring it to 1.0.0).